src/main/java/de/tum/in/www1/artemis/web/rest/TutorParticipationResource.java

```diff
@@ -0,0 +1,172 @@
+        return ResponseEntity.created(new URI("/api/exercises/" + exerciseId + "tutorParticipations/" + tutorParticipation.getId()))
+            .body(tutorParticipation);
+    }
+
+    /**
+     * POST /exercises/:exerciseId/tutorParticipations/:participationId/exampleSubmission: add an example submission to the tutor participation
+     *
+     * @param exerciseId      the id of the exercise of the tutorParticipations
+     * @return the ResponseEntity with status 200 (OK) and with body the exercise, or with status 404 (Not Found)
+     */
+    @PostMapping(value = "/exercises/{exerciseId}/exampleSubmission")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<TutorParticipation> addExampleSubmission(@PathVariable Long exerciseId, @RequestBody ExampleSubmission exampleSubmission) throws URISyntaxException {
+        log.debug("REST request to add example submission to exercise id : {}", exerciseId);
+        Exercise exercise = this.exerciseService.findOneLoadTutorParticipations(exerciseId);
+        Course course = exercise.getCourse();
+
+        if (!courseService.userHasAtLeastTAPermissions(course)) {
+            return forbidden();
+        }
+
+        User user = userService.getUserWithGroupsAndAuthorities();
+
+        TutorParticipation existingTutorParticipation = exercise.getTutorParticipationForExercise(user);
+        // Do not trust the user input
+        Optional<ExampleSubmission> exampleSubmissionFromDatabase = this.exampleSubmissionService.get(exampleSubmission.getId());
+
+        if (existingTutorParticipation == null || !exampleSubmissionFromDatabase.isPresent()) {
+            return ResponseEntity.notFound().build();
+        }
+
+        ExampleSubmission originalExampleSubmission = exampleSubmissionFromDatabase.get();
+
+        if (existingTutorParticipation.getStatus() != TutorParticipationStatus.REVIEWED_INSTRUCTIONS) {
+            return ResponseEntity.badRequest().headers(HeaderUtil.createFailureAlert("tutorParticipations", "tutorParticipationInWrongStatus", "You cannot assess an example submission if you haven't read the grading instructions yet.")).body(null);
+        }
+
+        // Check if it is a tutorial or not
+        boolean isTutorial = originalExampleSubmission.isUsedForTutorial();
+
+        // If it is not a tutorial we check the assessment
+        if (!isTutorial) {
+            // Retrieve the example feedback created by the instructor
+            List<Feedback> existingFeedback = this.exampleSubmissionService.getFeedbacksForExampleSubmission(exampleSubmission.getId());
+
+            // Check if the result is the same
+            // TODO: at the moment we check only the score +/10%, maybe we want to do something smarter?
+            float instructorScore = calculateTotalScore(existingFeedback);
+            float lowerInstructorScore = instructorScore - instructorScore / 10;
```

Comment:
Can we put this number (i.e. 10) into a constant?

Reply:
done